.drone.yml

```diff
@@ -89,32 +117,34 @@ trigger:
     - tag
   branch:
     - main
-   
+
 steps:
-  - name: deploy to cs-dev
-    image: quay.io/ukhomeofficedigital/kd
+  - name: checkout
+    image: quay.io/ukhomeofficedigital/hocs-helm-kubectl:1.0.0
     commands:
-      - cd kube
-      - ./deploy.sh
+      - git fetch --tags
+      - git checkout ${DRONE_TAG}
+      - git submodule update --init --recursive
+
+  - &deploy
+    name: cs-dev
+    image: quay.io/ukhomeofficedigital/hocs-helm-kubectl:1.0.0
     environment:
-      ENVIRONMENT: cs-dev
-      KUBE_TOKEN:
-        from_secret: hocs_templates_cs_dev
+      CHART_NAME: hocs-templates
+      KUBE_CLUSTER: acp-notprod
+      KUBE_NAMESPACE: cs-dev
```

Comment:
You could use [`${DRONE_STEP_NAME}`](https://docs.drone.io/pipeline/environment/reference/drone-step-name/) here to pull the value of KUBE_NAMESPACE from `name:` and save you having to define it twice in each anchor alias.

Reply:
I tried it in here and it didn't work as in `KUBE_NAMESPACE: ${DRONE_STEP_NAME},` it'll certainly work in the script because I saw you did it in the templates project (maybe) but I don't want to tie the script into drone.